### DISCUSSION TITLE
コンディショナルロールにノート投稿数についての条件があったらさらに嬉しいです

### DISCUSSION THREAD

**Discussion by tetsuya-ki** [misskey-dev/misskey#10225] [Ideas]
便利に使わせていただいています。
タイトルに書いた通りですが、コンディショナルロールの条件にノート投稿数について〜以上、〜以下のようなものがあったらなおのこと嬉しいと思いました。
(初心者であることを強調するため、アイコンを置きたいと考えており、それの条件に入れたい気持ちがありました)

素晴らしいソフトウェアを公開いただき、ありがとうございます。

**Comment by omi-key** [comment]
#10402 (https://github.com/misskey-dev/misskey/pull/10402/commits/41d1b1b8c994b2268e4e39d7e46fcbf7807c1ad0) で解消していそうです

**Comment by tetsuya-ki** [comment]
ありがとうございます。解決済みにします。
